services/user/Branding/ui/src/App.tsx

```diff
@@ -1,11 +1,12 @@
 import { useEffect, useRef, useState } from "react";
 
-import { Button } from "@shadcn/button";
-import { Label } from "@shadcn/label";
-import { Input } from "@shadcn/input";
+import { getSupervisor, siblingUrl } from "@psibase/common-lib";
 
-import { siblingUrl, getSupervisor } from "@psibase/common-lib";
-import { Nav } from "@components/nav";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
```

Comment:
Perhaps, given you're making this a model, we add a @/components/ui/index.js so we can import all 3 in one line?

Reply:
I'm seeing articles discussing barrel files (those index.ts files that combine everything) and they're raising questions about whether they're an antipattern or not, or in what cases, etc. I want to learn more about that before we combine these and export them. But something to look into for sure.